src/test/resources/jsonld/HT014215912

```diff
@@ -81,7 +81,7 @@
       "label" : "Geschichte",
       "type" : [ "SubjectHeading" ]
     } ],
-    "label" : "Karl, I. (747-814), Heiliges Römisches Reich, Kaiser | Herrscherbild | Geschichte ",
+    "label" : "Karl, I., Heiliges Römisches Reich, Kaiser | Herrscherbild | Geschichte ",
```

Comment:
Not sure if the deletion of the life-dates are intended.

Reply:
They weren't but now they are ;) This label should contain the same as the labels from the `ComplexSubject` because it is shown in the frontend. Also it's the same as the GND-label.